vignettes/load-get.Rmd

```diff
@@ -22,10 +22,8 @@ In general, every workflow starts with loading a simulation by calling the `load
 library(ospsuite)
 
 simFilePath <- file.path(getwd(), "..", "tests", "data", "Aciclovir.pkml", fsep = .Platform$file.sep)
-print(simFilePath)
 
 sim <- loadSimulation(simFilePath)
-print(sim)
```

Comment:
In the vignette it looked like this:
```
simFilePath <- file.path(getwd(), "..", "tests", "data", "Aciclovir.pkml", fsep = .Platform$file.sep)
print(simFilePath)
#> [1] "C:/Users/appveyor/AppData/Local/Temp/1/RtmpmKgCQQ/Rbuild4dc68be628d/ospsuite/vignettes/../tests/data/Aciclovir.pkml"

```
Here and below I removed the print statements for 2 reasons:
* The path produced on appveyour looks weird for the end users
* It always breaks the automated spell checking when I convert and integrate vignettes into the OSP online help, because parts of the paths were interpreted as words (and they are always different so I cannot add them to the exclusion list)
```
Misspelled words:
<htmlcontent> load-get.md: html>body>pre
--------------------------------------------------------------------------------
Rbuild
RtmpmKgCQQ
--------------------------------------------------------------------------------
```

Reply:
Hmm, ok